c7n/resources/airflow.py

```diff
@@ -0,0 +1,114 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+from c7n.manager import resources
+from c7n.query import QueryResourceManager, TypeInfo
+from c7n.filters.kms import KmsRelatedFilter
+from c7n.tags import RemoveTag, Tag, TagDelayedAction, TagActionFilter
+
+
+@resources.register('airflow')
+class ApacheAirflow(QueryResourceManager):
+    class resource_type(TypeInfo):
+        service = 'mwaa'
+        id = name = 'Name'
+        enum_spec = ('list_environments', 'Environments', None)
+        detail_spec = ('get_environment', 'Name', None, 'Environment')
+        arn = 'Arn'
+        arn_type = 'environment'
+        cfn_type = 'AWS::MWAA::Environment'
+
+    permissions = (
```

Comment:
Looks like the permissions should be as per the docs: https://docs.aws.amazon.com/mwaa/latest/userguide/access-policies.html
```
airflow:ListEnvironments,
airflow:GetEnvironment
```